spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,24 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'queues a job to lock the given account' do
+      other_user = FactoryGirl.create :user
+      expect(other_user).to receive(:lock_account!)
+      allow(User).to receive(:find).and_return(other_user)
+
+      post :lock_account, id: other_user.id
+    end
+  end
```

Comment:
Did you actually run your tests? They're failing: https://travis-ci.org/diaspora/diaspora/jobs/51284331#L440

See https://wiki.diasporafoundation.org/Testing_workflow

I'd say don't mock the call out, put an expectation like `.expect(other_user.reload.access_locked?).to be_truthy` after the post.